sshmenu/sshmenu.py

```diff
@@ -1,20 +1,27 @@
+import argparse
 import json
 import os
 import readchar
 import sys
 import time
+import shutil
 
 from subprocess import call
 from clint import resources
 from clint.textui import puts, colored
 
 
 def main():
+    # Check arguments
+    parser = argparse.ArgumentParser(prog='sshmenu', description='A simple tool for connecting to remote hosts via ssh.')
```

Comment:
I was thinking "use clint!" but clint's argument parsing is... lacking.

Anyways, I think the description can be more clear. I like `A convenient tool for bookmarking hosts and connecting to them via ssh.`, although it's somewhat wordy.